src/utils.jl

```diff
@@ -11,3 +11,16 @@ maybe_real(x::Complex) = iszero(imag(x)) ? real(x) : x
 
 maybe_only(x) = x
 maybe_only(x::Tuple{T}) where {T} = only(x)
+
+#Find the edge in the list of edges which involves v1 and v2
+function find_edge(edges::Vector{NamedDimEdge{Tuple}}, v1::Tuple, v2::Tuple)
```

Comment:
I guess this isn't needed anymore?

Reply:
Yes I believe this is now redundant so I have taken it out.